packages/manager/apps/vrack-services/src/pages/overview/index.tsx

```diff
@@ -71,15 +71,15 @@ export default function OverviewTab() {
         <OsdsMessage
           type={ODS_MESSAGE_TYPE.error}
           removable
-          onOdsRemoveClick={hideError}
+          onOdsRemoveClick={() => hideError()}
```

Comment:
```suggestion
          onOdsRemoveClick={hideError}
```
No need to create a new function